Wikipedia/Code/DiffListChangeCell.swift

```diff
@@ -35,6 +35,9 @@ class DiffListChangeCell: UICollectionViewCell {
     weak var delegate: DiffListChangeCellDelegate?
     
     func update(_ viewModel: DiffListChangeViewModel) {
+
+        textStackView.isAccessibilityElement = false
```

Comment:
I also notice there's also a `DiffListUneditedCell` that could maybe use some accessibility suppression. I can't remember how that cell is used though.

Reply:
I'm also not sure where it is used, but I suppressed it anyway